closed/OpenJ9.gmk

```diff
@@ -493,7 +493,7 @@ else
 endif
 
 ifeq (true,$(OPENJ9_ENABLE_CUDA))
-  CMAKE_ARGS += -DJ9VM_OPT_CUDA=ON
+  CMAKE_ARGS += -DJ9VM_OPT_CUDA=ON -DOMR_CUDA_HOME="$(CUDA_HOME)"
   CMAKE_CUDA_ENV := CUDA_BIN_PATH="$(CUDA_HOME)"
```

Comment:
We don't need `CMAKE_CUDA_ENV` any more: can you remove its definitions and uses?

Reply:
I was thinking, In order to keep the repository working, that could come in a later PR. So we'd accept this PR and it's counterpart in jdk11, then accept the OMR patch, then I'd open two new PRs cleaning up OpenJ9.gmk. That way, there's no window where cmake builds are broken. Does that sound OK?